server/main.cpp

```diff
@@ -69,9 +70,28 @@ int main(int argc, char *argv[])
     QCoreApplication a(argc, argv);
     a.setOrganizationName(QStringLiteral("TelegramQt"));
     a.setApplicationName(QStringLiteral("TelegramQt Server"));
+
     Telegram::initialize();
 
-    Config config;
+    QCommandLineParser parser;
+    parser.addHelpOption();
+
+    QCommandLineOption configFileOption(
```

Comment:
IMO those QStringList({QStringLiteral(), QStringLiteral()}), QStringLiteral(), QStringLiteral()) quite hard to read.
Please use the simpler constructor and separate setters.
I also think that it is more common to use lower cased 'C' for short "config" option. E.g. [nginx](http://nginx.org/en/docs/switches.html) use it. Capital short keys usually used if there are many other options and the lower-case one is already in use.

References:
- https://wiki.qt.io/API_Design_Principles#Avoiding_Common_Traps
- https://github.com/Kaffeine/telegram-qt/blob/c41256f6c2c132063eefbaf4fedba681e956deac/generator/main.cpp#L773

Reply:
:ok_hand: